### PR TITLE
shared config package

### LINKED ISSUE #7555: Shared network config
Right now, all of our apps have their own "config" for networks:
* [unlock-app](https://github.com/unlock-protocol/unlock/blob/master/unlock-app/src/config.js#L58)
* [paywall](https://github.com/unlock-protocol/unlock/blob/master/paywall/src/paywall-script/networkConfigs.ts)
* [locksmith](https://github.com/unlock-protocol/unlock/blob/master/locksmith/src/networks.ts)
* [unlock-js](https://github.com/unlock-protocol/unlock/blob/master/unlock-js/examples/networks.js)

Now that we have a sane way of doing packages, we should isolate that logic in its own package that's imported by other applications!

### PR DESCRIPTION
**Pull request by clemsos**
# Description

A shared package containing info about the different networks where Unlock is deployed

# Issues

Fixes #7555 

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Review by julien51 — APPROVED** [review]
Do u want to add the import inside the packages in another PR?

File: packages/networks/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
+{
```
Comment (julien51) [inline packages/networks/tsconfig.json:1]: I wonder if this should be shared at the root of the repo? wdyt?

Comment (clemsos) [reply packages/networks/tsconfig.json:1]: yep the best way is to have a `ts-config` package that we can import in the different workspaces. I think we keep it local for now then will add a package in another PR

**Comment by clemsos** [comment]
> Do u want to add the import inside the packages in another PR?

hmmm makes more sense to have imports inside this one so we can run the tests against the new config

**Pull request merged**